examples/README.md

```diff
@@ -0,0 +1,33 @@
+## Compiling ##
```

Comment:
Let's omit this as this is for advanced/dev and point directly to terraform registry here: 
https://registry.terraform.io/providers/chmurakrajowa/ochk/latest

```
terraform {
  required_providers {
    ochk = {
      source = "chmurakrajowa/ochk"
      version = "<version>"
    }
  }
}

provider "ochk" {
  # Configuration options
}